### PR TITLE
Bugfix: KeyList was cleared too late

### PR DESCRIPTION
**Pull request by bbuerger**
The capture is taking some time, therefor it is better to do the _keyList.Clear() a bit earlier, otherwise some keystrokes are missed

### CODE REVIEW

File: ScreenToGif/Windows/Recorder.xaml.cs [hunk 1]
```diff
@@ -698,21 +698,22 @@ private async void NormalAsync_Elapsed(object sender, EventArgs e)
             if (_stopRequested)
                 return;
 
-            _captureTask = _capture.CaptureAsync(new FrameInfo(_recordClicked, _keyList));
-            FrameCount = await _captureTask;
-
+            var frame = new FrameInfo(_recordClicked, _keyList);
             _keyList.Clear();
+            _captureTask = _capture.CaptureAsync(frame));
```
Comment (NickeManarin) [inline ScreenToGif/Windows/Recorder.xaml.cs:703]: There's an additional ) in this line.

**Comment by NickeManarin** [comment]
I got to test this PR just now. Fix the little issue and I'll test further. :)

**Comment by bbuerger** [comment]
Oh sorry, I forgot a commit and pushed an old version.
 I'm not working with git on a daily basis, I hope it works now.

**Comment by NickeManarin** [comment]
Thanks.

**Pull request merged**